src/App.tsx

```diff
@@ -1,14 +1,74 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID, getTodos } from './api/todos';
+import { TodoList } from './Components/TodoList/TodoList';
+import { Todo } from './types/Todo';
+import { Footer } from './Components/Footer/Footer';
+import { StatesOfFilter } from './types/StatesOfFilter';
+
+function getFilteredTodos(todos: Todo[], filterOption: StatesOfFilter): Todo[] {
+  let filteredTodos = [...todos];
+
+  if (filterOption !== StatesOfFilter.All) {
+    filteredTodos = filteredTodos.filter(todo => {
+      switch (filterOption) {
+        case StatesOfFilter.Active:
+          return !todo.completed;
+
+        case StatesOfFilter.Competed:
+          return todo.completed;
+
+        default:
+          return;
+      }
+    });
+  }
+
+  return filteredTodos;
+}
+
+function getCountOfActiveTodos(todos: Todo[]): number {
+  const countOfActiveTodos = todos.reduce((acc, todo) => {
+    if (!todo.completed) {
+      return acc + 1;
+    }
+
+    return acc;
+  }, 0);
+
+  return countOfActiveTodos;
+}
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<StatesOfFilter>(StatesOfFilter.All);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+
+  useEffect(() => {
+    getTodos()
+      .then(setTodos)
+      .catch(() => {
+        setErrorMessage('Unable to load todos');
+        setTimeout(() => {
+          setErrorMessage('');
+        }, 3000);
```

Comment:
Create a handler that sets an error message and then removes it in 3 seconds. As you have to do it every time when you fetch something